test/buyback/YieldReserve.t.sol

```diff
@@ -403,6 +392,21 @@ contract SwapTest is YieldReserveTest {
         vm.stopPrank();
     }
 
+    function test_RevertWhen_invalidSourceToken() public {
+        uint256 amt = 10;
+        IERC20(USDS).approve(address(yieldReserve), amt * USDsPrecision);
```

Comment:
Removed this line as it was not needed, same for `test_RevertWhen_invalidDestinationToken`